app/config/parameters.yml.dist

```diff
@@ -19,3 +19,5 @@ parameters:
     secret:            SuperSecretTokenPleaseChangeIt
     swp_updater.version.class: "SWP\WebRendererBundle\Version\Version"
     active_theme: "theme_1"
+    swp_multi_tenancy.tenant.class: SWP\MultiTenancyBundle\Model\TenantInterface
```

Comment:
I can put that directly into the config, seems more reasonable to put it into the parameters to me tho